compiler/luci/requires.cmake

```diff
@@ -1,3 +1,4 @@
+require("common-artifacts")
```

Comment:
common-artifacts uses luci. we cannot apply this change.

Reply:
@seanshpark Oh right. I first thought that since `common-artifacts` uses `luci` only when optimizing(`circle2circle`), this luci-test also uses what `common-artifacts` uses like recipe, `tfl-chef`(recipe -> tflite), `tflite2circle`(tflite -> circle).

But, this PR makes us have to add `common-artifacts` to preset, which looks bad:)